pylero.spec

```diff
@@ -1,28 +1,16 @@
-%define name pylero
-%define version 0.0.3
-%define unmangled_version 0.0.3
-%define release 1
-%define _unpackaged_files_terminate_build 0
-
 Summary: Python SDK for Polarion
-Name: %{name}
-Version: %{version}
-Release: %{release}
+Name: pylero
```

Comment:
- if we aren't using vars then in every release we might want to change these values at every required place
- any reason not defining vars?

Reply:
It's duplicate defining for spec, the var %{name} usage is still exist in the spec at following lines which use the current line value. 
For other vars, it's not been used or usage is problematic for Fedora spec, details in bug:
https://bugzilla.redhat.com/show_bug.cgi?id=2115102